src/content/pages/index.html

```diff
@@ -44,6 +44,22 @@ <h1 class="section-title">{{ site.homepage.title }}</h1>
                     <a href="{{ links.getStarted }}" target="_blank" class="c-btn c-btn--primary c-btn--large">{{ site.shared.get_started }}</a>
                     <a href="{{ links.donate }}" class="c-btn c-btn--secondary c-btn--large donate-link">{{ site.shared.become_a_sponsor }}</a>
                 </div>
+
+                {% set highly_sponsors = sponsors.platinum | concat(sponsors.gold) %}
+
+                <div class="eslint-highly-sponsors">
+                    <div class="sponsor-text">{{ site.homepage.sponsor.text }}</div>
+                    <div class="sponsors">
+                        {% for sponsor in highly_sponsors %}
```

Comment:
Should we actually set a limit here? because currently we have 2 platinum and 2 gold sponsors. If that increases then we might actually want to handle overflow also? we anyways have `more sponsors` cta?

Reply:
We want to show all platinum and gold sponsors, so we should have a way to have this keep growing.